lib/App/tldr.pm

```diff
@@ -31,12 +33,30 @@ sub parse_options {
         "h|help"    => sub { $self->_help },
         "o|os=s@"   => \($self->{platform}),
         "v|version" => sub { printf "%s %s\n", ref $self, $self->VERSION; exit },
+        "pager=s"   => \my $pager,
+        "no-pager"  => \my $no_pager,
     or exit(2);
     $self->{argv} = \@ARGV;
+    if (!$no_pager and -t STDOUT and my $guess = $self->_guess_pager($pager)) {
+        $self->{pager} = $guess;
+    }
     push @{$self->{platform}}, $self->_guess_platform, "common";
     $self;
 }
 
+sub _guess_pager {
+    my $self = shift;
+
+    my $cmd;
+    for my $try (grep $_, @_, $ENV{PAGER}, "less", "more") {
```

Comment:
This changes the default behaviour to always use pager.  That works fine for me!  But what if someone wants no-pager by default?  Perhaps remove `less` and `more` from the list, or add a configuration option?  I am trying to think ahead.

Reply:
I think it is OK to use pager by default.